articles/azure-functions/functions-reference-powershell.md

```diff
@@ -232,25 +238,28 @@ HTTP and webhook triggers and HTTP output bindings use request and response obje
 
 The Request object that's passed into the script comes from a type called `HttpRequestContext` which has the following properties:
 
-| Property  | Description                                                    |
-|-----------|----------------------------------------------------------------|
-| _Body_    | An object that contains the body of the request.               |
-| _Headers_ | A dictionary that contains the request headers.                |
-| _Method_  | The HTTP method of the request.                                |
-| _Params_  | An object that contains the routing parameters of the request. |
-| _Query_   | An object that contains the query parameters.                  |
-| _Url_     | The URL of the request.                                        |
+| Property  | Description                                                    | Type                      |
+|-----------|----------------------------------------------------------------|---------------------------|
+| _Body_    | An object that contains the body of the request.               | object                    |
```

Comment:
We just take an arbitrary object and serialize to json?  If so, perhaps mention the serialization.

Reply:
added a note